### PR TITLE
CF-364 Verify migration of dns_nameservers in subnets.

### PR DESCRIPTION
**Pull request by plomakin**
Created subnet with dns_nameservers attribute in source cloud.
Verified dns_nameservers attribute migrated to destination.
Refactored params validation in test_resource_migration.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3060/

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferry_devlab.tests.testcases.test_resource_migration
cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py:20: 
[E0401(import-error), ] Unable to import 'fabric.api'
cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py:21: 
[E0401(import-error), ] Unable to import 'fabric.network'

File: cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py [hunk 1]
```diff
@@ -567,18 +543,16 @@ def test_migrate_vms_parameters(self):
...
 
         src_vms = [vm for vm in src_vms if vm.status != 'ERROR']
 
-        self.validate_resource_parameter_in_dst(
-            src_vms, dst_vms, resource_name='VM', parameter='name')
-        self.validate_resource_parameter_in_dst(
-            src_vms, dst_vms, resource_name='VM', parameter='config_drive')
-        self.validate_resource_parameter_in_dst(
-            src_vms, dst_vms, resource_name='VM', parameter='key_name')
+        for param in vm_parameters:
```
Comment (roman-verchikov) [inline cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py]: maybe simply `for param in ['name', 'config_drive', 'key_name']:`?

File: cloudferry_devlab/cloudferry_devlab/tests/config.py [hunk 2]
```diff
@@ -417,7 +417,8 @@
          {'cidr': '10.4.2.0/24', 'ip_version': 4, 'name': 'subnet_1',
           'connect_to_ext_router': True, 'routers_to_connect': ['ext_router']},
          {'cidr': '10.9.2.0/24', 'ip_version': 4, 'name': 'subnet_2',
-          'connect_to_ext_router': True, 'routers_to_connect': ['ext_router']}]
+          "dns_nameservers": ["10.9.2.1"], 'connect_to_ext_router': True,
```
Comment (roman-verchikov) [inline cloudferry_devlab/cloudferry_devlab/tests/config.py]: I'd suggest that you add documentation regarding this particular test

File: cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py (hunk 1 above)
Comment (plomakin) [inline cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py]: ok, will fix that

**Comment by roman-verchikov** [comment]
Looks ok, would be good to have documentation describing test scenario

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
$PULL_STATUS
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3091/

**Comment by MirantisWorkloadMobility** [comment]
$PULL_STATUS
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3094/

**Comment by MirantisWorkloadMobility** [comment]
$PULL_STATUS
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3095/

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
:+1:

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**